framework/db/pgsql/Schema.php

```diff
@@ -468,7 +468,7 @@ protected function findColumns($table)
                     $column->defaultValue = bindec(trim($column->defaultValue, 'B\''));
                 } elseif (preg_match("/^'(.*?)'::/", $column->defaultValue, $matches)) {
                     $column->defaultValue = $matches[1];
-                } elseif (preg_match('/^(.*?)::/', $column->defaultValue, $matches)) {
+                } elseif (preg_match('/^(?:\()?(.*?)(?(1)\))(?:::.+)?$/', $column->defaultValue, $matches)) {
```

Comment:
just for me to fully understand this: why did you add the part that requires text after `::`? are you sure that can not be empty?